pkg/operator/certrotation/cabundle.go

```diff
@@ -41,7 +41,7 @@ type CABundleConfigMap struct {
 	EventRecorder events.Recorder
 }
 
-func (c CABundleConfigMap) EnsureConfigMapCABundle(ctx context.Context, signingCertKeyPair *crypto.CA) ([]*x509.Certificate, error) {
+func (c CABundleConfigMap) EnsureConfigMapCABundle(ctx context.Context, signingCertKeyPair *crypto.CA, signingCertKeyPairLocation string) ([]*x509.Certificate, error) {
```

Comment:
silly question while going through the breaking change in CEO, isn't that simply `ca.Namespace/ca.Name` at this point?

Reply:
That would be a location of the signer used to build this CA, so its `secret.Namespace/secret.Name`